inc/wp-forms-api.js

```diff
@@ -124,8 +124,8 @@
 			prepare: function() {
 				var data = this.model.toJSON();
 
-				data.input_name = this.input_name;
-				data.input_type = this.input_type;
+				data.input_name = this.model.get('input_name');
```

Comment:
Ohhhhhh.... This isn't actually necessary any more since those values are stored in the model, and they'll come out in `this.model.toJSON()`, so you can kill these two lines.